Tests/Acceptance/Backend/ContentDefenderCest.php

```diff
@@ -47,6 +47,7 @@ public function doNotSeeNotAllowedContentElementsInNewContentElementWizard(Backe
         $I->click('Content', '#element-tt_content-1 [data-colpos="1-200"]');
         $I->switchToIFrame();
         $I->waitForElement('.modal-dialog');
+        $I->wait(0.5);
```

Comment:
Rather do `$I->waitForText('Header Only')` instead of hardocded amount of seconds